pkg/store/cache/memcached.go

```diff
@@ -65,6 +72,9 @@ func NewRemoteIndexCache(logger log.Logger, cacheClient cacheutil.RemoteCacheCli
 // asynchronously stored in the cache.
 func (c *RemoteIndexCache) StorePostings(blockID ulid.ULID, l labels.Label, v []byte) {
 	key := cacheKey{blockID.String(), cacheKeyPostings(l)}.string()
```

Comment:
Could you please move this functionality into `cacheKeyPostings` and then make corresponding changes to `cacheKey`?

Reply:
Done